WebApiCoreSeed.Data.Tests/EF/DatabaseSeedTest.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.EF;
+using Xunit;
+
+namespace WebApiCoreSeed.Data.Tests.EF
+{
+    public class DatabaseSeedTest
+    {
+        [Fact]
+        public async Task Initialize_ShouldCreateAnUser()
```

Comment:
This is an exception to the rule, so you have a typo on `AnUser`. Should be `AUser` if understood correctly. I think it's because of how it sounds.

Reply:
ahh, I understand, I just read something about this, is the way that the second word it's pronounced, it doesn't matter if is consonant nor vocal, in this case, "you-ser", so, you're right, it's an a. Thank you